vendor/github.com/fluent/fluent-logger-golang/fluent/fluent.go

```diff
@@ -508,6 +520,15 @@ func (f *Fluent) run(ctx context.Context) {
 				return
 			}
 
+			if f.AsyncReconnectInterval > 0 {
```

Comment:
Per my comment above;

> So perhaps we should also have acceptable minimal and maximal intervals?

I'm actually wondering if the fluentd code itself should also define a minimum to prevent a close retry loop. Perhaps worth raising an issue for that and/or a PR to add at require a minimum value? 🤔

Reply:
> I'm actually wondering if the fluentd code itself should also define a minimum to prevent a close retry loop. Perhaps worth raising an issue for that and/or a PR to add at require a minimum value? 

I considered that when I submitted the PR there but thought I'd leave it to fluent maintainers to suggest values if they believed it necessary. I do also think that it can be left to downstream users to decide for themselves what those safe values are. I totally see it being appropriate in this project. I think 100ms/10s are fair min/maxes. We can go with that to start with and a second reviewer (I believe those are necessary for most/external changes) can perhaps suggest an alternative if they feel differently.

Reply:
> I think 100ms/10s are fair min/maxes. We can go with that to start with 

Agreed; we can always relax the range if users complain (and if there's valid reasons to have shorter or longer intervals)

> and a second reviewer (I believe those are necessary for most/external changes) can perhaps suggest an alternative if they feel differently.

Yes, general rule is to have a minimum of two reviewers (irregardless of who opened the PR) to reduce the risk of things being missed. In some rare exceptions we may merge with one review (trusting the maintainer that merged to have outweighed the risks).